addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3635,7 +3634,19 @@ msgctxt "#844"
 msgid "Deactivate"
 msgstr ""
 
-#empty strings from id 845 to 849
+#. Message in a dialog when a user wants to delete a repeating timer
+#: xbmc/pvr/windows/GUIWindowPVRBase.cpp
+msgctxt "#845"
+msgid "Are you sure you want to delete this repeating timer, including all timers it has scheduled?"
```

Comment:
sure, the label would have to be changed everywhere ofc. If current phrasing is ok for natives, then go ahead and we'll change `repeating timer` and related labels in a separate PR